components/service/glean/src/main/java/mozilla/components/service/glean/private/TimespanMetricType.kt

```diff
@@ -121,6 +121,16 @@ data class TimespanMetricType(
             return
         }
 
+        if (timerId != null) {
```

Comment:
The function description comment may need to be edited now.  I'm not sure the "Care should be taken..." bit really applies with this protection added.  Maybe replace it with a note about the new (correct) behavior?